windows.go

```diff
@@ -18,4 +18,12 @@
 
 package edgedb
 
+import "crypto/x509"
+
 var defaultHosts = []string{"localhost"}
+
+func getCertPool() (*x509.CertPool, error) {
+	// x509.SystemCertPool() doesn't work on Windows.
+	// https://github.com/golang/go/issues/16736
+	return x509.NewCertPool(), nil
```

Comment:
I think we should bundle the Mozilla CA bundle for parity with other bindings. certifi has a [copy](https://github.com/certifi/python-certifi/blob/master/certifi/cacert.pem)

Reply:
changed